.github/workflows/GPU.yml

```diff
@@ -0,0 +1,43 @@
+name: GPU
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: [self-hosted, GPU, benchmark]
+
+    steps:
+      - name: cleanup
+        run: rm -rf *
```

Comment:
Is that actually needed?

Reply:
I thought so, or how do I make sure the environment does not persist?

Reply:
If I do not do it, it checks out into a dir that already exists and throws an error